week-1/Homework/mandatory/4-tax.js

```diff
@@ -17,7 +19,13 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function formatCurrency() {}
+function formatCurrency(num) {
+  num = num*1.20;
+  num = num.toFixed(2)
+
+  return "£"+num.toString(); 
+
```

Comment:
Your fix is correct, the idea was to apply DRY more. So instead of 
```num=num*1.20 //you're using this in the previous function ```
you can use 
``` num=calculatetax(num) // or better even storing it in different variable```
```var taxedAmount= calculatetax(num) ```